utils/merkle-tree/src/lib.rs

```diff
@@ -30,40 +30,73 @@
 //! does not perform any sorting of the input data (leaves) nor when inner nodes are created.
 //!
 //! If the number of leaves is not even, last leaf (hash of) is promoted to the upper layer.
+#[cfg(not(feature = "std"))]
+extern crate alloc;
+#[cfg(not(feature = "std"))]
+use alloc::vec;
+#[cfg(not(feature = "std"))]
+use alloc::vec::Vec;
+
+/// Supported hashing output size.
+///
+/// The size is restricted to 32 bytes to allow for a more optimised implementation.
+pub type Hash = [u8; 32];
```

Comment:
This doesn't make any sense? What does "more optimized implementation" mean? You only use this for the `Keccak256` hasher below.

Reply:
Removed